### DISCUSSION TITLE
Brainstorm about interactions between throwable entities and the world

### DISCUSSION THREAD

**Discussion by JoeyDP** [ScicraftLearn/Minelabs#483] [Ideas]
There can be 'fun' interactions when for example an electron hits blocks or entities (shock, damage, remove grass from dirt, ...)

**Comment by MajaVerstraeten** [comment]
For starters, its is valuable to implement the "penetration power".

The penetration power actually makes the distinction between alpha radiation, beta radiation and gamma radiation (this is historical bullshit naming)
>> https://www.dreamstime.com/types-radiation-penetrating-power-paper-human-aluminum-lead-water-types-radiation-penetrating-power-image167415975

alpha == Helium nucleus: already stopped by paper
beta-  == electron: pass through paper, pass through human body, stopped by aluminium
beta+ == positron: idem
gamma == actual radiation of photons: even pass through aluminium, stopped by lead

Some other particles
neutrons: even pass through lead, stopepd by water.
Neutrino's: hardly interact at all. They can fly through the earth without making a single collision.

Some more ideas on what particles can do:

If electrons are shot through a gass, they can ionize the gass, by pinching off electrons from the atoms. When other electrons fall into the void that the electron left, a photon of a specific wavelength is emitted. 

If electrons or protons are shot at metal, they can charge the metal.

If positrons are shot through a player, a PET scan can be made

If alpha particles (helium nuclei) are shot through gold foil, you make the experiment of Rutherford

If atomic nuclei are shot through a magnetic field you make a mass-spectrometer. The path of the nucleus will be a circle with the radius dependent on the mass. https://gfycat.com/jauntyellipticaliceblueredtopzebra

If protons are shot towards each other, you have made a particle collider (such as in CERN)

.....
